config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -508,3 +508,28 @@ periodics:
       - ../kubernetes/vendor/k8s.io/apimachinery/...
       - ../kubernetes/vendor/k8s.io/apiserver/...
       - ../kubernetes/vendor/k8s.io/client-go/...
+
+- name: ci-build-and-push-k8s-at-golang-tip
+  interval: 24h
```

Comment:
Let's change it to 8h, I see that golang repo is quite busy as of now. We can reduce the frequency when they reach freeze

Reply:
Done.